descriptors/ModuleDescriptor-template.json

```diff
@@ -458,6 +458,34 @@
       "version": "1.0",
       "interfaceType": "system",
       "handlers": [
+        {
+          "methods": [
+            "POST"
+          ],
+          "pathPattern": "/circulation/expired-session-processing",
```

Comment:
I think it could help maintainability to name this endpoint with what it tries to achieve. What processing of expired sessions is done?

Reply:
@marcjohnson-kint , thank you for review, renamed